### ISSUE TITLE
Wrong message clicking on craftSign on craft piloted by different player

### ISSUE THREAD

**Issue by goodroach** [APDevTeam/Movecraft#587]
**Describe the bug**
The title says it all, think there is a bug with the logic because I remembered there being a specific message for that. 

**To Reproduce**
1. Have someone else pilot a craft for you
2. Try to click on their craft sign
3. "You are already commanding a craft!"

**Expected behavior**
The message should say something else, maybe something like a generic "Craft is already commanded!" or a more specific "Someone else is commanding this craft!"

**Versions (please complete the following information):**
 - Movecraft: [8.0.0 Beta 1]
 - Addons: [Not sure if this matters]
 - Java [JDK 16]
 - Minecraft 1.16.5

**Additional context**
This should probably be on the backburner but I'm writing this down so we are aware of this issue.

**Comment by TylerS1066** [comment]
Good idea, it should probably be a simple fix to `CraftSign.java`, the issue is that we always return that response regardless of who is the pilot around line 106.
